app/assets/javascripts/geoshiz.js

```diff
@@ -35,4 +36,4 @@
 
 $(function() {
 	coaster_enable_geolocation();
-});
\ No newline at end of file
+});
```

Comment:
This is still valid: updating to a PATCH verb, however, it would be preferable to keep the CSRF protection on this call. Rails now has some pretty useful ajax helpers.